cmd/dvotecli/commands/genesis.go

```diff
@@ -125,12 +140,48 @@ func genesisGen(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	prettyHeader("Genesis JSON")
-	fmt.Printf("%s\n ", data)
+
+	if printJson, _ := cmd.Flags().GetBool("json"); printJson {
+		bytes, err := json.MarshalIndent(keys, "", "  ")
+		if err != nil {
+			return err
+		}
+		fmt.Printf("%s\n", bytes)
+	} else {
+		prettyPrintKeyring(keys)
+	}
+
+	fmt.Println()
+
+	if file, _ := cmd.Flags().GetString("file"); file != "" {
+		if err := os.WriteFile(file, data.Bytes(), 0o600); err != nil {
+			return err
+		}
+	} else {
+		prettyHeader("Genesis JSON")
```

Comment:
Maybe we can avoid this header so the raw output JSON can be piped to tools like jq or ijq

Reply:
nice one! done